SETUP.md

```diff
@@ -70,7 +70,10 @@ Next, push your cloned manuscript:
 ```sh
 git push --set-upstream origin master
 ```
-
+An alternative way to push your cloned manuscrit would be:
+```sh
+git push --set-upstream https://github.com/OWNER/REPO
```

Comment:
This command is equivalent to `git push --set-upstream origin` (without `master`) assuming you used the https `set-url` command above.

Note you're missing the `$` to denote OWNER and REPO are variables.

Reply:
For clarification, `origin` refers to the location of the repository on GitHub. Therefore, `https://github.com/OWNER/REPO` and `origin` are synonymous. The convention is to set the remote location of the repository using a name like `origin`, and then using that to refer to that remote in commands.